ui/src/adapters/api/credentials.ts

```diff
@@ -143,21 +143,13 @@ export const linkStatusParser = getStrictParser<LinkStatus>()(
   z.union([z.literal("active"), z.literal("inactive"), z.literal("exceeded")])
 );
 
-const linkAttributes = getStrictParser<LinkAttributes>()(
```

Comment:
Why are you removing the Link attributes in this PR?

Reply:
Because it's not used and has changed in the api

Reply:
Adri updated it's local env and was getting parsing errors in the links list because the `attributes` prop is now gone.